test/WebSites/BasicWebSite/Areas/Area1/Views/RemoteAttribute_Home/Details.cshtml

```diff
@@ -64,13 +61,5 @@
 </div>
 
 <p>
-    @Html.ActionLink("Create a new one", "Create") |
-    @if (string.Equals("Aria", areaName, StringComparison.OrdinalIgnoreCase))
-    {
-        @Html.RouteLink("Go back to home", "Home")
-    }
-    else
-    {
-        @Html.RouteLink("Go to Aria home", "AriaHome")
-    }
+    @Html.ActionLink("Create a new one", "Create") | @Html.RouteLink("Go to Area1 home", "AriaHome")
```

Comment:
`DisplayNameFor()` and `DisplayFor()` don't "generate links". I'm missing something but this is a side conversation. Please restore the existing code and scenarios, except where the migration breaks things (e.g. namespaces) or you're doing straightforward cleanup.